### PR TITLE
[fix] used Syndesis#ensureImagePull for Camel K

### PR DESCRIPTION
**Pull request by mmuzikar**
//test: `@camel-k`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Camel-k Runtime | Extensions shouldn't be visible 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmuzikar** [comment]
WTH

**Comment by mmuzikar** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Camel-k Runtime | Extensions shouldn't be visible 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Checkstyle verification failed with:
```
utilities/src/main/java/io/syndesis/qe/resource/impl/
  CamelK.java:11:8: Unused import - io.syndesis.qe.test.InfraFail. [UnusedImports]
utilities/src/main/java/io/syndesis/qe/resource/impl/
  CamelK.java:30:8: Unused import - java.util.concurrent.TimeoutException. [UnusedImports]
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Camel-k Runtime | Extensions shouldn't be visible 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmuzikar** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmuzikar** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Camel-k Runtime | Extensions shouldn't be visible 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert 
Connection - CRUD | Create and delete 
Connection - CRUD | Kebab menu 
Connection - CRUD | Kebab menu edit and view 
Integration - DB to DB | Read and update operations 
Camel-k Runtime | Extensions shouldn't be visible 
Integration info | Check error for integration with the same name 
Integration info | Check error for integration which has same name after update 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Camel-k Runtime | Extensions shouldn't be visible 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmuzikar** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Camel-k Runtime | Extensions shouldn't be visible 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by jsafarik** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: utilities/src/main/java/io/syndesis/qe/resource/impl/CamelK.java [hunk 1]
```diff
@@ -146,27 +145,23 @@ private void installViaBinary() {
...
             arguments.append(" -n ").append(TestConfiguration.openShiftNamespace());
+            arguments.append(" --config ").append(OpenShiftUtils.binary().getOcConfigPath());
 
             final String command = LOCAL_ARCHIVE_EXTRACT_DIRECTORY + "/kamel install" + arguments.toString();
             log.info("Invoking " + command);
-            Runtime.getRuntime().exec(command).waitFor();
+            Process process = Runtime.getRuntime().exec(command);
+            process.waitFor();
+            if (process.exitValue() != 0) {
+                InfraFail.fail("The docker command installing Camel K failed. The exit value is " + process.exitValue() +
```
Comment (mkralik3) [inline utilities/src/main/java/io/syndesis/qe/resource/impl/CamelK.java]: Does the command use docker? If not I would delete docker mention from the log.

Comment (mmuzikar) [reply utilities/src/main/java/io/syndesis/qe/resource/impl/CamelK.java]: good point :)

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**